templates/postgres/base/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgres
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+      - name: postgres
+        image: postgres:15
```

Comment:
This is currently deployed to the same namespace as everything right? from the architecture diagram I thought you could deploy it on a different namespace.